fabric-registry-sync-v0/src/client/java/net/fabricmc/fabric/impl/client/registry/sync/RegistryRemovalChecker.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.client.registry.sync;
```

Comment:
Why client only? I think it would make sense to save this file on the server, thus you get the warning when taking a server world to the client.

We decided not to show the error on the server, but maybe a warning log (but still try to load) could be helpful?